frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -14,7 +14,8 @@
  * limitations under the License.
  */
 
-import React, { Component } from 'react';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
 import { getAccountsByKeystore, isAmbiguousName } from '@/routes/account/utils';
 import { route } from '@/utils/route';
```

Comment:
now that it is a functional component, please change to useNavigate hook instead of route util